Purchases/Public/RCPurchases.m

```diff
@@ -586,7 +586,7 @@ - (void)        purchaseProduct:(SKProduct *)product
     } else if (payment.productIdentifier) {
         productIdentifier = payment.productIdentifier;
     } else {
-        RCLog(@"makePurchase - Could not purchase SKProduct. Couldn't find its product identifier. This is possibly an App Store quirk.");
+        [RCLog info:[NSString stringWithFormat:@"%@", RCStrings.purchase.could_not_purchase_product_id_not_found]];
```

Comment:
this is the only change aside from expanding macros - we missed this string when moving stuff to `RCStrings` originally, so I added it now.